### DISCUSSION TITLE
Add one more curve about AWGN input_BER vs SNR in FEC figure ?

### DISCUSSION THREAD

**Discussion by WuShengHan** [NVlabs/sionna#167] [General]
Thank you for providing the Sionna code environment. Can I suggest to add a curve into the graph in the Evolution_of_FEC or some like example environment to describe the curve of input error rate generated by AWGN input against SNR? Thank you.

**Comment by SebastianCa** [comment]
Hi,

Thanks for the suggestion. We plan to integrate uncoded BERs into the `Evolution_of_FEC` notebook in the next release of Sionna.

**Comment by WuShengHan** [comment]
Hi, thanks a lot for your update about FEC input curve.

I also focus on FEC desgin like this

![未命名](https://github.com/NVlabs/sionna/assets/56661918/12344898-4e47-4daa-91e4-43dd8f13859b)


Are you interested in setting up the arena for some promising FEC cadidates?

A good FEC will let system no need Iterative_Detection_and_Decoding and Bit_Interleaved_Coded_Modulation

**Reply by SebastianCa** [reply]
Hi,
We're keen to showcase Sionna-related projects. Please refer to our "[Made with Sionna](https://nvlabs.github.io/sionna/made_with_sionna.html.)" section.
To feature your project, provide a link to the paper and the associated Sionna-based source code. A Jupyter notebook would also be appreciated.
